src/Eloquent/Model.php

```diff
@@ -52,30 +51,6 @@ abstract class Model extends BaseModel
      */
     protected $parentRelation;
 
-    /**
-     * Custom accessor for the model's id.
-     *
-     * @param  mixed  $value
-     * @return mixed
-     */
-    public function getIdAttribute($value = null)
-    {
-        // If we don't have a value for 'id', we will use the MongoDB '_id' value.
```

Comment:
This is a feature that needs to be kept: Allow to use a custom 'id' field as id.